sql/indicator_functions.sql

```diff
@@ -677,11 +676,12 @@ FROM
 WHERE
     patientGenderIs(pat.patient_id, p_gender) AND
     patientAgeWhenRegisteredForHivProgramIsBetween(pat.patient_id, p_startAge, p_endAge, p_includeEndAge) AND
-    patientHasStartedARVTreatmentDuringReportingPeriod(pat.patient_id, p_startDate, p_endDate) AND
-    patientPickedARVDrugDuringReportingPeriod(pat.patient_id, p_startDate, p_endDate, 0) AND
+    patientHasStartedARVTreatmentBeforeExtendedEndDate(pat.patient_id, p_endDate, 3) AND
+    patientWasOnARVTreatmentAtEndReportingPeriod(pat.patient_id, p_endDate) AND
+    mostRecentNotDocumentedViralLoadExamIsBelow(pat.patient_id, p_endDate, 1000) AND
```

Comment:
Hi @tmvumbi2 ,  I think you missed to add  "ARV drug order that has not ended before the reporting quarter end date"
condition. Please add this condition.